Userland/Libraries/LibJS/Tests/builtins/Intl/Segmenter/Segmenter.js

```diff
@@ -0,0 +1,13 @@
+describe("errors", () => {
+    test("called without new", () => {
+        expect(() => {
+            Intl.Segmenter();
+        }).toThrowWithMessage(TypeError, "Intl.Segmenter constructor must be called with 'new'");
+    });
+});
+
+describe("normal behavior", () => {
+    test("length is 0", () => {
+        expect(Intl.Segmenter).toHaveLength(0);
```

Comment:
These tests are a bit lacking given this commit also implements the full constructor. Generally the Intl constructor tests try to hit every exceptional case in the "errors" section, and every valid option in the "normal behavior" section. Mostly a copy-paste job from existing Intl tests.

Reply:
Fixed